### PR TITLE
Completed prep-css-flexbox

### PR DESCRIPTION
**Pull request by ayshahussain141**
Screenshot 2022-09-23 222958
Screenshot 2022-09-23 223648
Screenshot 2022-09-23 222514
Screenshot 2022-09-23 224438

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div{
+      display: flex;
+    }
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: For this assignment we don't want to make all divs display-flex, just the #card-deck

File: prep-css-flexbox/align-items.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    #card-deck{
+      flex-basis: calc(100%/3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: flex-basis is only meant for "flex items" so you'll want to use it in a rule set for `.card-wrapper`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Aysha, please see my notes. For this assignment we want to use flex-basis only on "flex items", so that property will need to be moved to a rule set for `.card-wrapper`

**Comment by ayshahussain141** [comment]
Screenshot 2022-09-24 190208
Screenshot 2022-09-24 190020
Screenshot 2022-09-24 190146
Screenshot 2022-09-24 185800

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**